query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/csw/persist/UpdateCswConfigurationSpec.groovy

```diff
@@ -169,6 +169,38 @@ class UpdateCswConfigurationTest extends Specification {
         report.messages().get(0).code == DefaultMessages.FAILED_PERSIST
     }
 
+    def 'Updating with flag password sends service properties without password'() {
+        setup:
+        updateCswConfiguration.setValue(createCswUpdateArgs(FLAG_PASSWORD))
+        serviceActions.read(_ as String) >> [(ID): TEST_SOURCENAME]
+        serviceReader.getServices(_, _) >> []
+        configurator.commit(_, _) >> mockReport(false)
+
+        def expectedUpdateConfig = getUpdateCswProps(FLAG_PASSWORD)
```

Comment:
I used the `*SourceUtils.configToServiceProps()` methods to create these which checks for the flag password and won't include it in the service properties to update if the password is equal to `FLAG_PASSWORD`. I originally opted to construct the expected map myself and not use those methods, but if this seems misleading I can switch it back to constructing the expected response myself. Coincidentally passing in `null` in here has the same effect as `FLAG_PASSWORD`